### DISCUSSION TITLE
The Common Voice 15 and Fleurs datasets are not evaluated in Chinese. I suspect that openAI is racist and culturally discriminatory. Such a trashy company

### DISCUSSION THREAD

**Discussion by dshwei** [openai/whisper#2456] [Q&A] [answered]
The Common Voice 15 and Fleurs datasets are not evaluated in Chinese. I suspect that openAI is racist and culturally discriminatory. Such a trashy company

**Comment by umbraclet16** [comment, marked as answer]
learn the word: mandarin
